dist/codecov.sh

```diff
@@ -119,73 +119,71 @@ cc_cli_args+=( $(write_existing_args CC_AUTO_LOAD_PARAMS_FROM) )
 cc_cli_args+=( $(write_existing_args CC_ENTERPRISE_URL) )
 cc_cli_args+=( $(write_existing_args CC_YML_PATH) )
 cc_cli_args+=( $(write_truthy_args CC_VERBOSE) )
-cc_cc_args=()
-cc_cc_args+=( $(write_truthy_args CC_FAIL_ON_ERROR) )
-cc_cc_args+=( $(write_existing_args CC_GIT_SERVICE) )
-cc_cc_args+=( $(write_existing_args CC_PARENT_SHA) )
-cc_cc_args+=( $(write_existing_args CC_PR) )
-cc_cc_args+=( $(write_existing_args CC_SHA) )
-cc_cc_args+=( $(write_existing_args CC_SLUG) )
-cc_create_report_args=()
-cc_cr_args+=( $(write_existing_args CC_CODE) )
-cc_cr_args+=( $(write_truthy_args CC_FAIL_ON_ERROR) )
-cc_cr_args+=( $(write_existing_args CC_GIT_SERVICE) )
-cc_cr_args+=( $(write_existing_args CC_PR) )
-cc_cr_args+=( $(write_existing_args CC_SHA) )
-cc_cr_args+=( $(write_existing_args CC_SLUG) )
-cc_du_args=()
-cc_du_args+=( $(write_existing_args CC_ENV) )
+cc_uc_args=()
```

Comment:
Please do not edit `dist/codecov.sh`. Run `install.sh` on your machine and only make changes in the `scripts` folder